src/v2/Apps/Partner/Routes/Articles/index.tsx

```diff
@@ -1,59 +1,107 @@
 import React from "react"
-import { Column, GridColumns, Box } from "@artsy/palette"
-import {
-  createPaginationContainer,
-  graphql,
-  RelayPaginationProp,
-} from "react-relay"
+import { Column, GridColumns, Box, Col, Row } from "@artsy/palette"
+import { createRefetchContainer, graphql, RelayRefetchProp } from "react-relay"
 import { Articles_partner } from "v2/__generated__/Articles_partner.graphql"
-import { ArticleCardFragmentContainer } from "../../Components/PartnerArticles/ArticleCard"
+import { ArticleCardFragmentContainer as ArticleCard } from "../../Components/PartnerArticles/ArticleCard"
+import { PaginationFragmentContainer as Pagination } from "v2/Components/Pagination"
```

Comment:
I know we used to do it in the past and it's all over the place but going forward we're avoiding renaming components like this. Just using the `FooFragmentContainer` form is fine/clearer

Reply:
ok, np

Reply:
fixed